packages/core/src/builder.class.ts

```diff
@@ -489,6 +491,14 @@ export class Builder {
     this.actions.push(action);
   }
 
+  static registerTrustedHost(host: string) {
+    this.trustedHosts.push(host);
+  }
+
+  static isTrusted(host: string) {
+    return this.trustedHosts.findIndex(allowedHost => host.indexOf(allowedHost) > -1) > -1;
```

Comment:
theres still an issue here, someone could register

maliciousbuilder.io and it works

or builder.io.malicious.com

it's important for host matching that the host is either exact, or `*.${host}`

Reply:
make sense, I'll update 👍